finch/src/services/email.js

```diff
@@ -0,0 +1,51 @@
+import nodemailer from 'nodemailer';
+
+import config from '../../config/index.js';
+
+const { emailSender, smtp } = config;
+
+export default class Email {
+
+  constructor() {
+    this.sender = emailSender;
+  }
+
+  init() {
+    this.transporter = nodemailer.createTransport({
+      host: smtp.host,
+      port: smtp.port,
+      auth: {
+        user: smtp.user,
+        pass: smtp.password,
+      },
+    });
+
+    return this;
+  }
+
+  setSender(senderEmail) {
+    this.from = senderEmail;
+    return this;
+  }
+
+  setReceiver(receiverEmail) {
+    this.to = receiverEmail;
+    return this;
+  }
+
+  setContent(bodyContent) {
+    this.html = bodyContent;
+    return this;
+  }
+
+  async send() {
+    const { transporter, from, to, html } = this;
```

Comment:
nitpick: probably its a good idea to throw an error here if one of these parameters don't exist.

Reply:
actually not a nitpick but a very good idea, this would save us from failing miserably while sending the email